File: wireless/rateplans/instance-get-example-1/instance-get-example-1.py
# Download the **Next-Gen** twilio-python library from:
# twilio.com/docs/libraries/python#installation-nextgen
from twilio.rest import Client

account_sid = "{{ account_sid }}" # Your Account SID from www.twilio.com/console
auth_token  = "{{ auth_token }}"  # Your Auth Token from www.twilio.com/console

client = Client(account_sid, auth_token)
wireless = client.preview.wireless

plan = wireless.rate_plans('WP467fb57a0aba9641a8209136d42545f8').fetch()
print(plan.sid)
print(plan.alias)

File: wireless/commands/instance-get-example-1/instance-get-example-1.js
// Download the **Next-Gen** twilio-node library from:
// twilio.com/docs/libraries/node#installation-nextgen
var accountSid = '{{ account_sid }}'; // Your Account SID from www.twilio.com/console
var authToken = '{{ auth_token }}';   // Your Auth Token from www.twilio.com/console

var twilioLibrary = require('twilio');
var client = new twilioLibrary.Twilio(accountSid, authToken);

client.preview.wireless.commands('DCcb6aa7c620874dd98ba6c9c4108d0ed0')
  .fetch(function(err, cmd) {
	console.log(cmd);
});
